src/object.js

```diff
@@ -57,22 +58,24 @@ export const unwind = _.curry((prop, x) =>
 // this one's _actually_ just like mongo's `$unwind`
 export const unwindArray = _.curry((prop, xs) => _.flatMap(unwind(prop))(xs))
 
-export const isFlatObject = overNone([_.isPlainObject, _.isArray])
```

Comment:
you removed this method

Reply:
This method was not part of the public API, is not used in spark and it's equivalent to `isTraversable`

Reply:
it was, it just wasn't in the readme. All exported methods are exported from index:

https://github.com/smartprocure/futil-js/blob/master/src/index.js#L7

Reply:
I mean it wasn't documented. Is it still a breaking change if it wasn't documented?